website/docs/docs/build/metricflow-commands.md

```diff
@@ -54,31 +39,34 @@ You can install [MetricFlow](https://github.com/dbt-labs/metricflow#getting-star
 
 **Note**, you'll need to manage versioning between dbt Core, your adapter, and MetricFlow.
 
-</TabItem>
+Something to note, MetricFlow `mf` commands return an error if you have a Metafont latex package installed. To run `mf` commands, uninstall the package.
 
+</TabItem>
 </Tabs>
 
-Something to note, MetricFlow `mf` commands return an error if you have a Metafont latex package installed. To run `mf` commands, uninstall the package.
-
 ## MetricFlow commands
 
 MetricFlow provides the following commands to retrieve metadata and query metrics. 
 
 <Tabs>
-<TabItem value="cloud" label="Commands for dbt Cloud CLI">
+<TabItem value="cloudcommands" label="Commands for dbt Cloud">
 
-You can use the `dbt sl` prefix before the command name to execute them in the dbt Cloud CLI. For example, to list all metrics, run `dbt sl list metrics`. For a complete list of the MetricFlow commands and flags, run the `dbt sl --help` command in your terminal.
+You can use the `dbt sl` prefix before the command name to execute them in the dbt Cloud IDE or dbt Cloud CLI. For example, to list all metrics, run `dbt sl list metrics`. For a complete list of the MetricFlow commands and flags, run the `dbt sl --help` command in your terminal. 
 
-- [`list`](#list) &mdash; Retrieves metadata values.
-- [`list metrics`](#list-metrics) &mdash; Lists metrics with dimensions.
-- [`list dimensions`](#list) &mdash; Lists unique dimensions for metrics.
-- [`list dimension-values`](#list-dimension-values) &mdash; List dimensions with metrics.
-- [`list entities`](#list-entities) &mdash; Lists all unique entities.
-- [`list saved-queries`](#list-saved-queries) &mdash; Lists available saved queries. Use the `--show-exports` flag to display each export listed under a saved query.
-- [`query`](#query) &mdash; Query metrics, saved queries, and dimensions you want to see in the command line interface. Refer to [query examples](#query-examples) to help you get started.
-- [`export`](#export) &mdash;  Runs exports for a singular saved query for testing and generating exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query.
-- [`export-all`](#export-all) &mdash; Runs exports for multiple saved queries at once, saving time and effort.
-- [`validate`](#validate) &mdash; Validates semantic model configurations.
+Note: Only the `list`, `query`, and `validate` commands are available in the dbt Cloud IDE.
+
+| <div style={{width:'250px'}}>Command</div>  | <div style={{width:'100px'}}>Description</div> | dbt Cloud IDE | dbt Cloud CLI |
+|---------|-------------|---------------|---------------|
+| [`list`](#list) | Retrieves metadata values. | ✅ | ✅ |
+| [`list metrics`](#list-metrics) | Lists metrics with dimensions. |  ✅ | ✅ |
+| [`list saved-queries`](#list-saved-queries) | Lists available saved queries. Use the `--show-exports` flag to display each export listed under a saved query. |  ✅ | ✅ |
```

Comment:
you can also do `--show-parameters` to show the full query parameters each saved query uses.